radicle-cli/src/commands/patch.rs

```diff
@@ -156,17 +171,9 @@ impl Args for Options {
                     "c" | "checkout" => op = Some(OperationName::Checkout),
                     unknown => anyhow::bail!("unknown operation '{}'", unknown),
                 },
-                Value(val) if op == Some(OperationName::Show) && patch_id.is_none() => {
-                    patch_id = Some(term::cob::parse_patch_id(val)?);
-                }
-                Value(val) if op == Some(OperationName::Update) && patch_id.is_none() => {
-                    patch_id = Some(term::cob::parse_patch_id(val)?);
-                }
-                Value(val) if op == Some(OperationName::Checkout) && patch_id.is_none() => {
-                    patch_id = Some(term::cob::parse_patch_id(val)?);
-                }
-                Value(val) if op == Some(OperationName::Delete) && patch_id.is_none() => {
-                    patch_id = Some(term::cob::parse_patch_id(val)?);
+                Value(val) if parse_oid_for_op(patch_id, &op) => {
+                    let val = string(&val);
+                    patch_id = Some(Rev::from(val));
```

Comment:
I think it was more readable before, as I now have to understand the `parse_oid_for_op` function

Reply:
I was trying to avoid code duplication here, maybe we can improve the name of the function? or do you want that I restore the previous code?

Reply:
If you want to de-duplicate the code, maybe it's just:

```rust
                Value(val) if !op.is_none() && !op == Some(OperationName::List) && patch_id.is_none() => {
```

Reply:
Oh now I get it, sorry I was not looking on I wrote the method! but make send, I will change it right now